SECURITY.md

```diff
@@ -0,0 +1,77 @@
+# Security Policy
+
+## Supported Versions
+
+Unless otherwise specified, we recommend to only use the most recent minor version release.
+
+## Reporting a Vulnerability
+
+**Please do not file a public ticket** mentioning the vulnerability.
+
+To disclose a vulnerability, please submit a `Security Advisory` via the `Security` tab on the impacted repository.
+
+If a repository doesn't have the proper security reporting set up, please email `security@celestia.org` to report the vulnerability.
+
+## Disclosure Policy
+
+Please first submit the vulnerability you discovered using the instructions in
+Reporting a Vulnerability. Once you have done so, you may share the details with
+third parties after either of the following, whichever is sooner:
+
+- the vulnerability has been fixed and the Celestia program owner has provided
+permission for you to disclose it; or
+- 120 days after you submit the vulnerability
+
+### Scope
+
+The scope of this security policy applies to the code repositories under the [Celestia Org Github](https://github.com/celestiaorg) and any related infrastructure.
```

Comment:
?

```suggestion
The scope of this security policy applies to the code repositories under the [Celestia Labs GitHub](https://github.com/celestiaorg) and any related infrastructure.
```

Reply:
@CelestiaOrg is not the Celestia Labs GitHub. It is the Celestia community GitHub.

Reply:
Ah then it should be
```suggestion
The scope of this security policy applies to the code repositories under the [Celestia community Github](https://github.com/celestiaorg) and any related infrastructure.
```

Reply:
IMO it should just be explicit and say the CelestiaOrg GitHub. There's no need to give CelestiaOrg an alias.